timetable/forms.py

```diff
@@ -160,3 +161,78 @@ def clean(self):
         #TODO check for encoding
         #TODO check for csv import errors
         return cleaned_data
+
+class SelectDateForm(Form):
+    date = Html5DateField(label=_('Date'), initial=get_next_schoolday)
+    
+    
+class AddDayPlanForm(Form):
+    date = Html5DateField(label=_('Date'), initial=get_next_schoolday())
+    i = 0
+    choices = list(tuple())
+    for name in Schedule.objects.all().iterator():
```

Comment:
Here is a chicken-egg problem

When starting fresh, to populate a database You need to process all the imports including this forms.py file and this code would be executed asking to retrieve all database objects. In other words when applying database migrations You need to have the tables in the database to create them.

To fix this You need to put this code inside a function and set choices to this function (not function value, but function object). An elegant lambda solution would look like:
```python3
choice = ChoiceField(choices=lambda: enumerate(Schedule.objects.all()), label=_('Choice'))
```

Consider using [ModelChoiceField](https://docs.djangoproject.com/en/2.2/ref/forms/fields/#modelchoicefield), which does all of this for us